edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/DockerConfig.cs

```diff
@@ -86,9 +100,29 @@ internal static CreateContainerParameters GetCreateOptions(string createOptions)
         }
 
         internal static string ValidateAndGetImage(string image)
+        {
+            return ValidateAndGetImage(image, new EnvironmentWrapper());
+        }
+
+        internal static string ValidateAndGetImage(string image, IEnvironmentWrapper env)
         {
             image = Preconditions.CheckNonWhiteSpace(image, nameof(image)).Trim();
+
+            Match match_host = ImageUpstreamRegex.Match(image);
```

Comment:
I think you used rust code style in C#, e.g match_host should be matchHost.  Normally we don't use _ character in variable name.

Reply:
I realized :). Thanks